### PR TITLE
Initial meaningful commit

### PR DESCRIPTION
**Pull request by ChrisCarini**

### CODE REVIEW

File: src/main/java/com/chriscarini/jetbrains/examplelocconfigurationplugin/LoCThresholdConfigurer.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+                new ArrayList<>(List.of(
+                    new ChangeThresholdTimeInfo("S", 10, 1, 2),
+                    new ChangeThresholdTimeInfo("M", 20, 3, 4),
+                    new ChangeThresholdTimeInfo("L", Integer.MAX_VALUE, 5, 6)
+                ))
+            );
+            changeThresholdService.setChangeThresholdIconInfo(
+                new ChangeThresholdIconInfo(100, 75, 50)
+            );
+            LoCService.getInstance(project).computeLoCInfo();
```
Comment (ChrisCarini) [inline src/main/java/com/chriscarini/jetbrains/examplelocconfigurationplugin/LoCThresholdConfigurer.java:43]: Here is the code in **this** plugin that reaches into [ChrisCarini/loc-change-count-detector-jetbrains-plugin](https://github.com/ChrisCarini/loc-change-count-detector-jetbrains-plugin) and configures custom thresholds.

File: src/main/resources/META-INF/loc-cop-support.xml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<idea-plugin version="2">
+  <extensions defaultExtensionNs="com.intellij">
+    <postStartupActivity implementation="com.chriscarini.jetbrains.examplelocconfigurationplugin.LoCThresholdConfigurer"/>
```
Comment (ChrisCarini) [inline src/main/resources/META-INF/loc-cop-support.xml:3]: The configuration of custom thresholds happens as a 'post startup activity', so it happens shortly after a project has been opened.

File: src/main/resources/META-INF/plugin.xml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    <name>Example LoC Configuration</name>
+    <vendor email="jetbrains@chriscarini.com" url="https://jetbrains.chriscarini.com">Chris Carini</vendor>
+
+    <!-- please see http://www.jetbrains.org/intellij/sdk/docs/basics/getting_started/plugin_compatibility.html
+         on how to target different products -->
+    <depends>com.intellij.modules.platform</depends>
+
+    <depends optional="true" config-file="loc-cop-support.xml">
+        com.chriscarini.jetbrains.loc-change-count-detector-jetbrains-plugin
+    </depends>
```
Comment (ChrisCarini) [inline src/main/resources/META-INF/plugin.xml:12]: Here we declare an optional dependency on the [ChrisCarini/loc-change-count-detector-jetbrains-plugin](https://github.com/ChrisCarini/loc-change-count-detector-jetbrains-plugin).

If a user **HAS** the LoC plugin and **this** plugin installed in their IDE, **this** plugin will register a 'post startup activity' and configure the LoC plugin.

If a user **DOES NOT HAVE** the LoC plugin installed in their IDE and **HAS** _this_ plugin installed in their IDE, then **this** plugin is essentially a `noop`, and does nothing.

**Review by bansalsulabh — APPROVED** [review]
lgtm

**Pull request merged**